src/main/java/com/rusticisoftware/cloud/v2/client/ApiClient.java

```diff
@@ -628,7 +622,7 @@ public File prepareDownloadFile(Response response) throws IOException {
    *
    * @param <T> Type
    * @param path The sub-path of the HTTP URL
-   * @param method The request method, one of "GET", "POST", "PUT", "HEAD" and "DELETE"
```

Comment:
Isn't `HEAD` still valid for some calls in V2? I guess we may not need this if we don't send a HEAD request in the API client 🤷‍♂